setup.py

```diff
@@ -94,10 +94,12 @@ def err(*args, **kwargs):
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
+        'Programming Language :: Python :: 3.6',
         'Topic :: Software Development',
         'Topic :: Scientific/Engineering',
         'Topic :: Utilities',
     ],
+    install_requires=install_requires,
```

Comment:
I didn't use easy_install directly. I was testing with a virtual environment but the pip coming with pyvenv-3.4 was broken. [The solution](http://askubuntu.com/questions/488529/pyvenv-3-4-error-returned-non-zero-exit-status-1/528625#528625) was to create a venv without pip and then install pip with `curl https://bootstrap.pypa.io/get-pip.py | python`. I do this often but I must have forgotten the second step once, that's how I ended up without pip. Users not having pip is still a real possibility, especially in Python 2.

Also, on my machine setup_tools/pip does not recognize if PyQt has been installed in an alternative way. For instance, install PyQt 5.6.0 with anaconda, and add `install_requires = ["PyQt5 >= 5.6.0"]` to argos' setup.py script. Pip will not recognize that PyQt has been installed. This is probably a bug in the PyQt install script, but for me another reason not to use `install_requires`. It is a source of errors and I can only use it for the mandatory dependencies (PyQt and Numpy). Argos also has many optional dependencies (PyQtGraph, h5py, NetCDF, etc). The users will have to install the optional dependencies explicitly anyways, so it's not to much extra effort to also install PyQt and Numpy by hand.

Hope this explains my motivation clearly.